docs/install.md

```diff
@@ -78,76 +78,75 @@ their accounts on the Pactus blockchain.
 To create a new wallet, run this command. The wallet will be encrypted by the
 provided password.
 
-```text
-./pactus-wallet ~/pactus/wallets/wallet_1 create
+```bash
+./pactus-wallet --path ~/pactus/wallets/wallet_1 create
```

Comment:
```suggestion
./pactus-wallet create --path ~/pactus/wallets/wallet_1
```

Reply:
All commands accept --path option before the subcommand. So let's keep it same

Reply:
OK